### DISCUSSION TITLE
Running “update” in LXC returns “command not found” error

### DISCUSSION THREAD

**Discussion by unxplained** [tteck/Proxmox#1900] [Help Needed] [answered]
Hi,

I’m trying to update my applications (zigbee2mqtt, home assistant etc)

Using the full command works (bash …..) but typing “update” in the LXC Console returns “command not found”. 

Is there something I missed?

Thanks

**Comment by tteck** [comment]
The "update" command only works for installations after this [commit](https://github.com/tteck/Proxmox/commit/e45aa5e42a98ecbc3c75bd034376fa2337113ab1) (Oct. 5th 2023)

**Reply by unxplained** [reply]
is there a way i get this command for older installations?

**Reply by tteck** [reply, marked as answer]
To manually integrate the `update` command, modify `<app>` to match the precise application name found in the ct folder of this repository, and then execute the command within the LXC console:
```bash
echo "bash -c \"\$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/<app>.sh)\"" >/usr/bin/update
chmod +x /usr/bin/update
```
Here's an example for Zigbee2MQTT:
```
echo "bash -c \"\$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/zigbee2mqtt.sh)\"" >/usr/bin/update
chmod +x /usr/bin/update
```
If the example command was executed in the Zigbee2MQTT LXC console, an `update` command would be created.

**Reply by unxplained** [reply]
Thank you!
